torchbenchmark/models/pytorch_unet/__init__.py

```diff
@@ -89,6 +88,7 @@ def jit_callback(self):
             self.model = torch.jit.script(self.model)
 
     def eval(self) -> Tuple[torch.Tensor]:
+        torch.backends.cudnn.deterministic = True
```

Comment:
This function is not being used by the downstream `benchmarks/dynamo/torchbench.py`. So I am wondering how this change will fix the CI issue in pytorch/pytorch? @bhack

Reply:
The problem is that more in general pytorch repo benchmark  is not considering deterministic meta filed in this repository so it is hard to understand what to do there.